apis/coursebatchcertificateapi/coursebatchcertificateapi.yaml

```diff
@@ -0,0 +1,455 @@
+            id:
+              description: The identifier for the signatory
+              type: string
+            designation:
+              description: The designation of the signatory
+              type: string
+              example: CEO
+            image:
+              description: Signatory id
+              type: string  
+      issuer:  
+        description: The filter for the template
+        type: object
+        properties:
+          name:
+            description: The issuer name
+            type: string
+          url:
+            description: The issuer URL
+            type: string
+          publicKey:
+            description: The public keys
+            type: array
+            items:
+              type: integer
+              format: int64
+            example: [7, 8]  
+    required:    
+    - identifier
+    - criteria
+  RemoveCertificateTemplateRequest:
+    title: RemoveCertificateTemplateRequest
+    type: object
+    properties:
+      request:
+        $ref: '#/definitions/RemoveCourseBatchTemplateRequest'
+    required:
+    - request
+  RemoveCourseBatchTemplateRequest:
+    title: RemoveCourseBatchTemplateRequest
+    type: object
+    properties:
+      batch:
+        $ref: '#/definitions/RemoveCourseBatchTemplate'
+    required:
+    - batch
+  RemoveCourseBatchTemplate:
+    title: RemoveCourseBatchTemplate
+    type: object
+    properties:
+      courseId:
+        description: The course identifier
+        type: string
+      batchId:
+        description: The batch identifier
+        type: string
+      template:
+        $ref: '#/definitions/RemoveBatchTemplate'
+    required:
+    - courseId
+    - batchId
+    - template
+  RemoveBatchTemplate:
+    title: RemoveBatchTemplate
+    type: object
+    properties:
+      templateId:
+        description: The template identifier
+        type: string
+    required:
+    - templateId
+  IssueCertificateRequest:
+    title: Issue certificate request
+    type: object
+    properties:
+      request:
+        $ref: '#/definitions/Request'
+    required:
+    - request
+  Request:
+    title: Request
+    type: object
+    properties:
+      courseId:
+        description: The course identifier
+        type: string
+      batchId:
+        description: The batch identifier
+        type: string
+      userIds:
+        description: The optional User IDs
+        type: array
+        items:
+          type: string
+    required:
+    - courseId
+    - batchId
+  IssueCertificateResponse:
+    description: Issue certificate Response
+    type: object
+    properties:
+      result:
+        description: The response details
+        type: object
+        properties:
+          courseId:
+            description: The course identifier
+            type: string
+          batchId:
+            description: The batch identifier
+            type: string
+          status:
+            description: The certificate issue status
```

Comment:
@rahul-tarento @maheshkumargangula please provide an example for certificate issue status

Reply:
example is provided